google/cloud/firestore_v1/base_client.py

```diff
@@ -137,7 +146,6 @@ def __init__(
         self._client_options = client_options
 
         self._database = database
-        self._emulator_host = os.getenv(_FIRESTORE_EMULATOR_HOST)
```

Comment:
where is this initialized now?

Reply:
Before the `__super__` call (line 126 above).